runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java

```diff
@@ -124,7 +124,7 @@ else if (!widget.getText().isEmpty())
 		PrayerType prayer = getActivePrayerType();
 		if (prayer == null)
 		{
-			resetTextColor(widget);
+			widget.setTextColor(config.getDefaultColor().getRGB());
```

Comment:
Null check?

Reply:
resetTextColor becomes unused here? and I think it is needed in order to reset changed colors due to the prayer flicking.